### ISSUE TITLE
example doesn't work on my post

### ISSUE THREAD

**Issue by Macadoshis** [ngx-translate/i18n-polyfill#13]
Hello,

I cloned the project and started "example/src/i18n" as-is, but it seems that the labels from html file are not translated.
Translation does actually work using I18n service (see chrome console).

![image](https://user-images.githubusercontent.com/2878861/35218491-2d007730-ff6f-11e7-964c-c3f16eaa1170.png)

I even added :
`<h2 i18n>another test ^_^</h2>` inside index.html, but this key doesn't get translated either.

![image](https://user-images.githubusercontent.com/2878861/35218554-5fdaf93c-ff6f-11e7-818a-a47532e3186c.png)

What am I missing ?

**Comment by JacobSiegle** [comment]
I don't believe the HTML is supposed to be translated because you're expected to use the built in angular i18n. Only the calls to the polyfill are translated because that's all this lib does.
